### ISSUE TITLE
how do train the models of pytorch

### ISSUE THREAD

**Issue by genty1314** [miaow1988/ShuffleNet_V2_pytorch_caffe#23]

**Comment by miaow1988** [comment]
Just use the office training codes with this network model.
Please check the official training codes: https://github.com/pytorch/examples/tree/master/imagenet
